src/libstd/ffi/c_str.rs

```diff
@@ -1040,7 +1040,8 @@ impl CStr {
     /// ```
     #[inline]
     #[stable(feature = "cstr_from_bytes", since = "1.10.0")]
-    pub unsafe fn from_bytes_with_nul_unchecked(bytes: &[u8]) -> &CStr {
+    #[rustc_const_unstable]
```

Comment:
you need to specify a feature name like in https://github.com/rust-lang/rust/blob/f45f52532a394d2d607fc5693364ad820049376d/src/test/ui/feature-gates/feature-gate-rustc_const_unstable.rs#L18